library/oid.c

```diff
@@ -152,7 +148,6 @@ int FN_NAME( ATTR1_TYPE ATTR1, ATTR2_TYPE ATTR2, const char **oid ,         \
     return( MBEDTLS_ERR_OID_NOT_FOUND );                                   \
 }
 
-#if defined(MBEDTLS_X509_USE_C) || defined(MBEDTLS_X509_CREATE_C)
```

Comment:
How much of a code size increase does this cause in a build with `MBEDTLS_OID_C` (for the sake of rsa or pkparse) but without X509?